app/assets/stylesheets/settings.scss

```diff
@@ -20,14 +20,6 @@
   .github-repo-row {
     background: var(--card-bg);
     padding: var(--su-3);
-
```

Comment:
Decided to drop this styling as a part of spring cleaning because we're going to turn github repos into extension anyway AND this been using some one-off variables sooo